duplex-messaging.md

```diff
@@ -0,0 +1,74 @@
+# SMP agent protocol - duplex communication over SMP protocol
+
+## Table of contents
+
+- [Abstract](#abstract)
+- [SMP agent commands](#smp-agent-commands)
+
+## Abstract
+
+Protocol describes the command that can be sent to the client that communicates with multiple SMP servers. This protocol allows to manage multiple simplex connections organised into groups defining duplex communication channels.
+
+## SMP agent commands
+
+Commands syntax below is provided using [ABNF][1].
+
+Each transmission between the user and SMP agent must have this format/syntax
+(after unwrapping it out of SMP message):
+
+```abnf
+transmission = (userCmd / agentMsg) CRLF
+userCmd = create / alias / subscribe
+          / invite / accept
+          / send / acknowledge
+          / status / info
+          / suspend / delete
+agentMsg = connection / connectionInfo / invitation / message / unsubscribed / ok / error
+
```

Comment:
maybe list all user commands first, then all agent messages after, in separate sections titled by comments? hard to keep track which you're reading right now

Reply:
I was going to do it as in simplex-messaging.md - just wanted to dump them together first